react/features/video-menu/components/native/ConnectionStatusComponent.js

```diff
@@ -425,10 +443,11 @@ class ConnectionStatusComponent extends Component<Props, State> {
  * @returns {Props}
  */
 function _mapStateToProps(state, ownProps) {
-    const { participantID } = ownProps;
+    const { initialStats, participantID } = ownProps;
 
     return {
         _bottomSheetStyles: ColorSchemeRegistry.get(state, 'BottomSheet'),
+        _initialStats: initialStats,
```

Comment:
you don't need to do this, it will be part of props because it's passed explicitly